docs/webhooks/Getting-Started.md

```diff
@@ -0,0 +1,56 @@
+---
+tags: [Webhooks]
+---
+
+# Getting Started
+
+## Tunnelling traffic to your local machine
+
+To test out webhooks locally without deploying an application to an environment or exposing your test environments to the Internet, you can use ngrok, a free service which tunnels internet traffic to your local environment. 
+
+Download ngrok [here](https://ngrok.com/download), then follow their getting started guide.
+
+## Configuring webhooks through the portal
+
+Once you’re up and running with ngrok, you’ll need to configure our webhooks as follows:
+
+1. Generate a secret in the portal and save its value for later
+
+  ![Generate a secret in the portal](../../assets/images/webhooks/Generate-Secret.png)
+
+2. Create subscription in the portal, using the HTTPS URL provided by ngrok, ensuring you specify the path that your application has been set up on e.g. `https://9999-123-456-789-12.ngrok.io/webhooks`
+
+  ![Create a subscription](../../assets/images/webhooks/Create-Subscription1.png)
+  
+  ![Create a subscription details](../../assets/images/webhooks/Create-Subscription2.png)
+
+Ensure ngrok is running and forwarding traffic to the port on which your local application is running.
+
+You can then send the test event by clicking the button in admin portal or sending an API request
+The request will be sent to your local application via ngrok, where you can:
+
+### 1. Handle the event
+
+Receive the POST request on your HTTPS endpoint.
+
+### 2. Check the signature
```

Comment:
Should we call out that this is for them to ensure the webhook request has come from WealthKernel?

Reply:
Alternatively shrink the section to just verify the webhook signature and link to the Secrets guides and put in more detail of to verify the signature there?